root/Readme.ja.md

```diff
@@ -1,64 +1,21 @@
-﻿# Open 棟梁
-Open 棟梁は、.NET Framework をベースとしたアプリケーション フレームワークです。  
-Open 棟梁のプログラムは、以下のリポジトリで公開しています。
-- OpenTouryoTemplates リポジトリ
-    - 概要  
-    Open 棟梁を使用したアプリケーションの**開発基盤 (テンプレート)** となるもの  
-    SQL Server を既定の DBMS とし、Visual Studio のバージョンごとにフォルダが分かれている
-    - 想定利用者  
-    Open 棟梁を使用してアプリケーションを開発する人
-- OpenTouryo リポジトリ (**現在のリポジトリ**)
-    - 概要  
-    OpenTouryoTemplates リポジトリの**母体**となるもの  
-    (Open 棟梁の各機能は、まず OpenTouryo リポジトリで開発され、その後 OpenTouryoTemplates リポジトリに取り込まれる)
-    - 想定利用者  
-    OSS 開発者
+# Open 棟梁テンプレート・ベース (Visual Studio 2015 用)
```

Comment:
@daisukenishino

This repository is *NOT* template base.
Please do not replace this file with Readme file of template base.